asv/plugins/snakeviz.py

```diff
@@ -1,10 +1,7 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 # -*- coding: utf-8 -*-
 
-from __future__ import absolute_import, division, unicode_literals, print_function
 
-from __future__ import (absolute_import, division, print_function,
-                        unicode_literals)
 
 
```

Comment:
Here we have 4 blank lines in a row after the change. The CI is not failing, because at the moment we're skipping some files like this one from checking the styles. That's alright for this PR, I'm going to merge it, but if you work on new ones, we could remove the line with `# -*- coding: utf-8 -*-` and all the blank lines. So, the beginning of the file would be:
```python
# Licensed under a 3-clause BSD style license - see LICENSE.rst
from .. import profiling
...
```

That would be the best in my opinion, as developers using this file would avoid the extra noise of the blank lines. When there are imports they useful to group the imports in blocks, but here they won't bring much value.